hypothesis_test.ipynb

```diff
@@ -4,7 +4,7 @@
   "metadata": {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #29.      expected_data = pd.DataFrame({&nbsp;
&nbsp; &nbsp; &nbsp; 'Code': [],
&nbsp; &nbsp; &nbsp; 'long': [],
&nbsp; &nbsp; &nbsp; 'lat': [],
&nbsp; &nbsp; &nbsp; 'fraud': [],
&nbsp; &nbsp; &nbsp; 'predicted_fraud': []
&nbsp; })
</pre><p>Why is this expected to be empty?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf_common/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Because there are no more than one measurement per sample, so we can't get a mean/variance from them. All of them are unique, so there's nothing to predict.

Reply:
All tree samples are == 1, so we can't get average/variance so we skip them in the t test.